android/app/src/main/java/com/zulipmobile/notifications/NotificationsModule.kt

```diff
@@ -0,0 +1,88 @@
+package com.zulipmobile.notifications
+
+import com.google.firebase.iid.FirebaseInstanceId
+import com.google.firebase.iid.InstanceIdResult
+import android.os.Bundle
+import android.util.Log
+import androidx.core.app.NotificationManagerCompat
+import com.facebook.react.bridge.*
+import com.google.android.gms.common.ConnectionResult
+import com.google.android.gms.common.GoogleApiAvailability
+import java.lang.Exception
+
+internal class NotificationsModule(reactContext: ReactApplicationContext?) :
```

Comment:
This should probably be non-nullable.

(That's the sort of thing the auto-conversion has to guess about, because that distinction doesn't exist in the Java type system.)